<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetWindow</class>
 <widget class="QMainWindow" name="AssetWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Análise do ativo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_ticker">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ticker</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_transactions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>1061</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_transactions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Transações</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_yields">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>355</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rendimentos</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_yields">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>1061</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
